libs/platform/src/lib/form/multi-combobox/commons/base-multi-combobox.ts

```diff
@@ -632,6 +633,21 @@ export abstract class BaseMultiCombobox extends CollectionBaseInput implements A
                     isInitDataSource = false;
                 }
 
+                const isSelectedShown = this.selectedShown$.getValue();
+
+                if (isSelectedShown && this._selectedSuggestions.length > 0) {
+                    const selectedSuggestionsLength = this._selectedSuggestions.length;
+
+                    for (let i = 0; i < selectedSuggestionsLength; i++) {
+                        const selectedSuggestion = this._selectedSuggestions[i];
+                        const idx = this._suggestions.findIndex((item) => equal(item.value, selectedSuggestion.value));
+
+                        if (idx !== -1) {
+                            this._suggestions[idx].selected = true;
+                        }
+                    }
+                }
```

Comment:
it can be slightly shortened 
```suggestion
                if (isSelectedShown && this._selectedSuggestions.length > 0) {
                    for (let i = 0; i < this._selectedSuggestions.length; i++) {
                        const selectedSuggestion = this._selectedSuggestions[i];
                        this._suggestions.find((item) => equal(item.value, selectedSuggestion.value))?.selected = true;
                    }
                }
```

Reply:
Hi @dmitry-stepanenko 
in this case, we will get ts error 
`The left-hand side of an assignment expression may not be an optional property access.ts(2779)`